docs/pages/branding/pricing.tsx

```diff
@@ -0,0 +1,707 @@
+      />
+      <Box
+        component="img"
+        src="/static/branding/block2-white.svg"
+        loading="lazy"
+        alt=""
+        sx={{
+          position: 'absolute',
+          bottom: -40,
+          left: 60,
+        }}
+      />
+      <Container>
+        <Typography variant="h2" sx={{ mb: { xs: 5, md: 10 }, maxWidth: 550 }}>
+          Here&apos;s <UnderlinedText>what to expect</UnderlinedText> from Material-UI
+        </Typography>
+        <Grid container spacing={5}>
+          <Grid item xs={12} md={6}>
+            <BrandingCard
+              color="emerald"
+              image="/static/branding/pricing/fast.svg"
+              imgProps={{ width: 28, height: 40 }}
+              title="Volume discount"
+            >
+              <Box sx={{ mt: 2 }}>
+                The licenses are on a per-developer basis. We offer the following tiered discounts
+                from list prices when purchasing more than one license for your development team:
+                <ul>
+                  <li>
+                    2-5 Licenses: <strong>10% discount</strong>
+                  </li>
+                  <li>
+                    6-10 Licenses: <strong>15% discount</strong>
+                  </li>
+                  <li>11+ License capped, extra developers do not need to be licensed.</li>
+                </ul>
+              </Box>
+            </BrandingCard>
+          </Grid>
+          <Grid item xs={12} md={6}>
+            <BrandingCard
+              color="vividBlue"
+              image="/static/branding/pricing/customizable.svg"
+              imgProps={{ width: 36, height: 36 }}
+              title="Perpetual license"
+            >
+              <Box sx={{ mt: 2 }}>
+                When you purchase,{' '}
+                <strong>
+                  you are granted a license to use a version of the product in perpetuity
+                </strong>
+                . There are no further charges unless you choose to extend your license to cover
+                newer versions.
+                <br />
+                <br />
+                Please note that while the use of the software is perpetual, support and corrective
+                maintenance are not. We do not provide issue resolution to versions older than 12
+                months. We roll bug fixes, performance enhancements, and other improvements into new
+                releases; we don&apos;t patch, fix or in any way alter older versions.
+              </Box>
+            </BrandingCard>
+          </Grid>
+          <Grid item xs={12} md={6}>
+            <BrandingCard
+              image="/static/branding/pricing/community.svg"
+              imgProps={{ width: 40, height: 40 }}
+              title="1-year subscription to new versions"
+            >
+              <Box sx={{ mt: 2 }}>
+                When you make a purchase{' '}
+                <strong>you get a subscription to license new versions for 366 days</strong>. You
```

Comment:
```suggestion
                <strong>you have the right to use any version released within the following year</strong>. You
```
(No need to mention the extra day of padding)